.github/workflows/release.yml

```
@@ -0,0 +1,87 @@
+name: Release packages and images
+
+on:
+  workflow_dispatch:
+  release:
+    types:
+      - published
+      - edited
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: npm install
+      - name: Publish package
+        run: npm publish
```

I despise useless names when the underlying command is a one-liner and is more descriptive. Convenient for debugging & as easy to understand, surely?

```suggestion
      - uses: actions/checkout@v2
      - run: npm install
      - run: npm publish
```